luwak/src/main/java/uk/co/flax/luwak/presearcher/TermFilteredPresearcher.java

```diff
@@ -132,7 +135,7 @@ public Query build() {
 
     public static final FieldType QUERYFIELDTYPE;
     static {
-        QUERYFIELDTYPE = new FieldType(TextField.TYPE_STORED);
+        QUERYFIELDTYPE = new FieldType(TextField.TYPE_NOT_STORED);
         QUERYFIELDTYPE.setIndexOptions(IndexOptions.DOCS_AND_FREQS_AND_POSITIONS_AND_OFFSETS);
```

Comment:
Someone internally raised this as a question, do we really **need** all these?
From what I can see, we could get away with DOCS_ONLY, for basic matching? To use debug/PresearcherMatchCollector, then offsets are needed as well, but would it be possible to configure that in/out?